classes/Config.js

```diff
@@ -4,12 +4,15 @@ const dotenv = require('dotenv').config()
 let config = {
 	DEFAULT_THEME: process.env.DEFAULT_THEME,
 	DEFAULT_ROOM: process.env.DEFAULT_ROOM,
+
 	DEFAULT_INLINE_PREVIEW: process.env.DEFAULT_INLINE_PREVIEW.toLowerCase() === 'true',
-	HTTP_PORT: parseInt(process.env.HTTP_PORT), // also used directly as .env in the bin/www file
-	HTTPS_PORT: parseInt(process.env.HTTPS_PORT), // also used directly as .env in the bin/www file
+
+	HTTP_PORT: parseInt(process.env.HTTP_PORT),
+	HTTPS_PORT: parseInt(process.env.HTTPS_PORT), 
+
 	SECRET_SALT: process.env.SECRET_SALT,
 	themes: [],
-	urlPrefix: '/', // not yet a variable
+	urlPrefix: '/'
```

Comment:
```suggestion
	urlPrefix: '/' // not yet a variable
```
I wanted to make it so that you can host the chat under `example.com/someRandomSubdirectory/`, and have it as a variable under `.env`, but this would cause many changes to be needed to make paths dynamic elsewhere in the codebase

Reply:
Yep, I could check this out sooner or later. I'm gonna be doing a discussion with all ideas for rewrite for mirating to the fastify ecosystem, Maybe get this working then.

Reply:
Even with fastify, I would leave the comment in so that in the future, we remember to bring this over to the `.env`

Reply:
Uhm, a suggestion!
Why dont we use `.env` and `config.ini`
For reference, I made a socket app for fun, and I used config, It's reliable for hosts and ports and stuff, while .env for password, salts and stuff.
Link: https://github.com/janaSunrise/ZeroCom 😅

Reply:
This seems like it's only going to complicate things, why not just use one and not the other?
I don't see the benefit in using two if we can do it with just one, and it'll probably confuse people

Reply:
I mean having `.ini` is good for public settings and `.env` for variables, in that way if people don't attach a `.env` we can run it normally without any issues. `.env` is usually for sensitive data, while .ini for config, and these are not mostly config. Hope you get it

Reply:
I see your point, yes, that would be good, you've changed my mind, it'd be great if and `.env` were optional and we could default to certain things, but make them modifiable in the `.env`.

Honestly, **a randomly generated salt should automatically just be created if it's not present in the `.env` or the file is missing.**

But it's a completely separate change and this PR has taken a bit long for us to get through, can you make a separate PR for that after this one is merged? Just to make it easier to manage, since there's already a lot of changes in this one PR